microsoft/testsuites/cpu/stress.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from __future__ import annotations
+
+from lisa import (
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from microsoft.testsuites.cpu.common import verify_cpu_hot_plug
+
+
+@TestSuiteMetadata(
+    area="cpu",
+    category="stress",
+    description="""
+    This test suite is used to run cpu related tests under stress.
+    """,
+)
+class CPUStressSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+            The test case take idle cpus into offline and online for 10 times.
```

Comment:
Specify the steps should refer to verify_cpu_hot_plug

Reply:
updated